UI fixes

jj7lin opened the pull request:
Changes:
-updated questionnaire, edit/view profile, and matching/swiping ui
-added new icons and logos to each page
-increased font size for input and output
-fixed distance issue and set max to 500km
-moved name and age from profile to colored-box container
![image](https://user-images.githubusercontent.com/72889234/99649173-dd4c8900-2a21-11eb-9290-6d0ea2018bb1.png)
![image](https://user-images.githubusercontent.com/72889234/99649223-eb020e80-2a21-11eb-98d4-f74a00c24fac.png)
![image](https://user-images.githubusercontent.com/72889234/99649281-04a35600-2a22-11eb-9f70-22a181b5a976.png)
![image](https://user-images.githubusercontent.com/72889234/99649305-0c62fa80-2a22-11eb-8aab-ebf4a8788078.png)

Updated UI after code review:
![image](https://user-images.githubusercontent.com/72889234/99703143-c1b6a200-2a64-11eb-95af-4bac13dfd0fc.png)
![image](https://user-images.githubusercontent.com/72889234/99703187-ced39100-2a64-11eb-85cb-0f355cf45411.png)
![image](https://user-images.githubusercontent.com/72889234/99703272-e9a60580-2a64-11eb-897d-66927e08fb42.png)

wclui committed:
fix merge conflict

wclui committed:
fix merge conflict

wclui committed:
Merge branch 'development' of https://github.com/echanlee/PP-friends into development

jj7lin committed:
css changes to multiple pages

added new background colors, icons, and logos to each page

jj7lin committed:
Merge branch 'development' into UI_fixes

chalynpai commented on src/Register.js:
```diff
@@ -74,39 +71,35 @@ class Register extends React.Component {
-                onChange={this.handleInputChange}
-              />
-              <br></br>
+        <img src="ppFriendsLogo.png"></img>
```
Remove logo - please refer to wireframes before adding things.

jj7lin replied:
won't be removing logos - some group members liked it as it adds more personality to each page :)

chalynpai commented on src/Questionnaire.js:
```diff
@@ -1,118 +1,124 @@
+        <div className="header">
+          <br></br>
+          <br></br>
+          <img src="question-penguin.svg"></img>
```
Please refer to the wireframe that was created and change design accordingly. 

![Survey](https://user-images.githubusercontent.com/48136962/99679483-fec16b00-2a49-11eb-98c2-c6ef77380b3b.png)

chalynpai replied:
you make the scroll through a set height and overflow: auto

jj7lin replied:
added scrolling feature, but left the radio buttons to the bottom of each question

mnsison replied:
good job joy!

chalynpai commented on src/Questionnaire.css:
```diff
@@ -1,8 +1,93 @@
+
+.boxed {
+  /*turqouise box section/background */
+  background-color: #4dadb7;
```
Change this colour to match the register + profile sections the code is FED987

jj7lin replied:
made it coral to match the register

chalynpai commented on src/Questionnaire.css:
```diff
@@ -1,8 +1,93 @@
+  text-align: center;
+}
+
+.Questions {
+  font-size: 30px;
+  font-weight: bold;
+}
```
Reduce weight of the answers. The bolded makes it difficult to read but the questions look good.

chalynpai commented on src/ViewProfile.js:
```diff
@@ -1,102 +1,113 @@
+      <div className="Profile">
+        <Header id={this.state.userId} />
+        <form id="profileForm">
+          <h1>View My Profile 👀</h1>
```
remove the emoji

chalynpai commented on src/ViewProfile.js:
```diff
@@ -1,102 +1,113 @@
-          <form id="profileForm">
-            <h1>View My Profile</h1>
-            <p>Name:</p>
+  render() {
```
For this section can you display the results next to each other instead of under?
For example:
Name: Joy 
Birthday: 22-02-1999
etc

chalynpai replied:
For all my comments on this please also do the same for edit profile

chalynpai commented on src/ViewProfile.js:
```diff
@@ -1,102 +1,113 @@
+        <Header id={this.state.userId} />
+        <form id="profileForm">
+          <h1>View My Profile 👀</h1>
+          <img src="ppFriendsLogo.png"></img>
```
Assume this is placement for when the profile photo will be. Can you move it to the left of the profile. Look at wireframes for reference.

chalynpai commented on src/SwipeProfiles.js:
```diff
@@ -150,48 +156,53 @@ class SwipeProfiles extends React.Component {
-        <Header id={this.state.id}/>
+        <Header id={this.state.id} />
         <br></br>
         <header class="pageTitle">Potential Friends!</header>
```
Move this nav bar to its own separate div or it will be effected by the "swipeprofile" styling. Move it to before the div above.

chalynpai replied:
Please make sure your other pages does this too. It's to ensure the nav bar sticks to the top of all the pages.

jj7lin replied:
theres no navbar code here?

chalynpai replied:
the header id = this.state.id is the nav bar (its calling the Header file) just move it before the <div className = "swipeProfile"> into its own div

chalynpai commented on src/Matches.js:
```diff
@@ -1,136 +1,151 @@
+          </button>
+        );
+      }
+      matchingSection = (
+        <h3 id="Matches-congrats">
+          <img src="happy-penguin.svg"></img>
+          <p>Congratulations, you have a match! </p>
+          <div className="matchingBox">
+            <br></br>
+            <h2>{userItems}</h2>
+            <br></br>
+          </div>
+        </h3>
```
Please note this section also relies on Melody's section. This should be done after her backend is completed so the styling can be done based on that. Please discuss with Melody.

mnsison commented on src/CreateProfile.js:
```diff
@@ -1,171 +1,214 @@
+      name: "",
+      birthday: "",
+      bio: "",
+      gender: "Female",
+      genderPreference: "Female",
```
can we change this to default as "Both"

wclui committed:
Merge branch 'development' of https://github.com/echanlee/PP-friends into development

jj7lin committed:
implemented changes as discussed in group chat and code review

Removed unnecessary punctuation, adjusted background color width for all pages to be the same size, updated input boxes to same width to remain consistency, updated questionnaire page to implement scrolling within the questionnaire container.

wclui committed:
merge conflict

wclui committed:
Merge branch 'development' of https://github.com/echanlee/PP-friends into development

wclui committed:
Merge branch 'development' into display_messaged_users

wclui committed:
fix merge conflict

jj7lin committed:
reverting back register page

echanlee commented:
- Make edges of the boxes consistent (all boxes should be rounded)
- Keep view profile and edit profile values on the same line. (similar to the pictures attached)
- Make input boxes rounded as well
- Fix merge conflicts 

![image](https://user-images.githubusercontent.com/40003439/99745012-cd2abd00-2aa6-11eb-92ba-79309b7e32ba.png)

jj7lin committed:
Merge branch 'development' into UI_fixes

mnsison commented on src/Header.css:
```diff
@@ -1,52 +1,61 @@
 }
 
 .div-header a:hover,
-.div-header a:active 
```
I'm not sure if this is important but is the change in colour supposed to be only for hover? There is a container/rectangle that appears once a link is selected/active (see attached image), but I don't know where it comes from.
![Screenshot 2020-11-20 140029](https://user-images.githubusercontent.com/35570430/99839504-cba9d500-2b38-11eb-91f0-8d39fff5120a.jpg)

jj7lin replied:
i think its for hover and active

wclui committed:
saving progress

wclui committed:
added view friend profile and seeing read and unread messages

jj7lin committed:
redid UI for profile pages

horizontal layout instead of vertical for the profile pages

jj7lin committed:
committing some additional design changes

committing my changes so a team member can work on a file i worked on
